### ISSUE TITLE
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found

### ISSUE THREAD

**Issue by wforumw** [browserpass/browserpass-native#117]
### General information

-   Operating system + version: Debian Bullseye
-   Browser + version: Google Chrome Version 98.0.4758.102 (Official Build) (64-bit)
-   Information about the host app:
    -   How did you install it?
downloaded a pre-built binary

-   Information about the browser extension:
    -   Browserpass extension version as reported by your browser: Version 3.7.2

### Exact steps to reproduce the problem

1. make BIN=browserpass-linux64 configure

2. sudo make BIN=browserpass-linux64 install

3. make hosts-chrome-user

4. Install extension in Google-Chrome 

### What should happen?

Extension should connect to host

### What happened instead?

If I click on the Browserpass symbol in the Browser I get
Error: Error: Native host has exited.

If I run the Browser from the terminal I see following
/usr/bin/browserpass-linux64: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/bin/browserpass-linux64)

**Comment by erayd** [comment]
@maximbaz Do you recall who the Debian packager is?

Which version of libc is the precompiled binary linked against? We may need to either bump it, or build a different version for compatibility with Debian.

**Comment by maximbaz** [comment]
I think @wforumw is using a pre-built binary, compiled by me. I want to look into whether it's possible to get rid of libc dependency from our binary, it would be nice...

**Comment by wforumw** [comment]
I am using the pre-built binary
If I run I get
```
$ /usr/bin/browserpass-linux64
/usr/bin/browserpass-linux64: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/bin/browserpass-linux64)
```
The version installed is 3.0.8

**Comment by maximbaz** [comment]
Should be fixed in 3.0.9, thanks, let me know if it's not the case!

**Comment by wforumw** [comment]
Thanks!!
Now it works
